src/jdk.jlink/share/classes/jdk/tools/jlink/internal/plugins/DefaultCompressPlugin.java

```diff
@@ -92,16 +93,29 @@ public void configure(Map<String, String> config) {
         if (level != null) {
             switch (level) {
                 case LEVEL_0:
+                    System.err.println(MessageFormat
+                            .format(getMessage("compress.warn.argumentdeprecated"), LEVEL_0));
```

Comment:
```suggestion
                    System.err.println(getMessage("compress.warn.argumentdeprecated", LEVEL_0));
```

same applies to L102 and 107.